### DISCUSSION TITLE
Sicurezza associazione dispositivi ZigBee

### DISCUSSION THREAD

**Discussion by fabrizioDO** [indomus/forum#2760] [ZigBee, Z-Wave, Thread, Matter]
Ciao a tutti,

ho qualche dubbio su come funzione ZigBee e sulla sicurezza.
Come ZigBee Coordinator ho Sonoff ZigBee 3.0 USB Dongle Plus v2 e su Home Assistant nell’elenco dei dispositivi disponibili nell’ integrazione MQTT mi sono ritrovato due dispositivi del mio vicino (Aqara vibration sensor e TRADFRI LED bulb).
Quindi chiunque abbia un coordinator nei paraggi può leggere i miei sensori?

Grazie

**Comment by snakuzzo** [comment]
No
